index.php

```diff
@@ -46,6 +46,36 @@
 define('NO_UPGRADE_CHECK', true);
 define('ABORT_AFTER_CONFIG', true);
 
+/**
+ * Checks if the command line maintenance mode has been enabled. Skip the config bootstrapping.
+ *
+ * @param string $configfile The relative path for config.php
+ * @return bool True if climaintenance.html is found.
+ */
+function checkclimaintenance($configfile) {
```

Comment:
Moodle standards allow underscore in function name, so you could use `check_climaintenance()` if you wanted.